docs/robots/ridgeback/tutorials_ridgeback.mdx

```diff
@@ -1,12 +1,1080 @@
+So while Gazebo won't be used with your real Ridgeback, RViz is used with both.
+
+You can use the following launch invocation to start RViz with a predefined configuration suitable for visualizing any standard Ridgeback config.
+
+```
+roslaunch ridgeback_viz view_robot.launch
+```
+
+You should see RViz appear.
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/ridgeback_images/ridgeback_rviz.png"
+      width="800"
+    />
+    <figcaption>Ridgeback RViz</figcaption>
+  </figure>
+</center>
+
+The RViz display only shows what the robot knows about its world, which presently, is nothing.
+Because the robot doesn't yet know about the barriers which exist in its Gazebo world, they are not shown here.
+
+#### Driving with Interactive Controller
+
+RViz will also show Ridgeback's interactive markers around your Ridgeback's model.
+These will appear as a blue ring and red arrows.
+Depending on your robot, there will also be green arrows.
+If you don't see them in your RViz display, select the Interact tool from the top toolbar and they should appear.
+
+Drag the red arrows in RViz to move in the linear X direction, and the blue circle to move in the angular Z direction.
+If your robot supports lateral/sideways movement, you can drag the green arrows to move in the linear Y direction.
+RViz shows you Ridgeback moving relative to its odometric frame, but it is also moving relative to the simulated world supplied by Gazebo.
+If you click over to the Gazebo window, you will see Ridgeback moving within its simulated world.
+Or, if you drive real Ridgeback using this method, it will have moved in the real world.
+
+#### Visualizing Sensors
+
+The RViz tool is capable of visualizing many common robotic sensors, as well as other data feeds which can give us clues as to what the robot is doing and why.
+A great place to start with this is adding the [LaserScan](http://wiki.ros.org/rviz/DisplayTypes/LaserScan) plugin to visualize the laser scans being produced
+by the simulated UST-10LX or LMS-111. In the left panel, click the "Add" button, then select the "Topics" tab, and then select the `front/scan` topic:
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/ridgeback_images/ridgeback_rviz_add_laser.png"
+      width="800"
+    />
+    <figcaption>Adding a laser scan visualization to Ridgeback</figcaption>
+  </figure>
+</center>
+
+Click "OK" and you should see laser scan points now visible in the RViz window, relative to the robot.
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/ridgeback_images/ridgeback_rviz_laser.png"
+      width="800"
+    />
+    <figcaption>Visualizing Ridgeback with simulated laser scans</figcaption>
+  </figure>
+</center>
+
+If you use the interactive markers to drive around, you'll notice that the laser scan points move a little bit but generally stay where they are.
+This is the first step toward map making using [Gmapping](#gmapping-demo).
```

Comment:
Should be  uppercase throughout our documentation: Gmapping

Based off ros.org,   http://wiki.ros.org/gmapping

There are other references in this documentation page to 'gmapping'

Reply:
Updated throughout